libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalAnalysisView.tsx

```diff
@@ -171,4 +179,16 @@ export class CausalAnalysisView extends React.PureComponent<
   ): ICausalAnalysisSingleData[] | undefined {
     return globalCausalEffects?.sort((d1, d2) => d2.point - d1.point);
   }
+
+  private getPolicyElement(): any {
+    if (!this.state.isCausalPolicyDataLoading) {
+      return (
+        <TreatmentView
+          data={this.state.currentGlobalCausalPolicy}
+          telemetryHook={this.props.telemetryHook}
+        />
+      );
+    }
+    return <LoadingSpinner label={localization.Common.loading} />;
```

Comment:
nit: we can have conditional operator here
`return !this.state.isCausalPolicyDataLoading ? (
      <TreatmentView
        data={this.state.currentGlobalCausalPolicy}
        telemetryHook={this.props.telemetryHook}
      />
    ): <LoadingSpinner label={localization.Common.loading} />;`

Reply:
The if statement has the same effect. Will leave the logic as is.